src/jsxstyle-components.js

```diff
@@ -0,0 +1,9 @@
+// Mostly copied from:
+// https://github.com/threepointone/glamor/blob/
+// 5e7d988211330b8e2fca5bb8da78e35051444efd/src/jsxstyle.js
+/*
+ * This is used to generate the helper functions
+ * like: glamorous.block and the <glamorous.Block /> elements
+ */
+
+export default ['view', 'block', 'inlineBlock', 'flex', 'row', 'column']
```

Comment:
Actually, because we're going to be constructing things ourselves (rather than using glamor's implementation) perhaps this should be more of a config:

```js
export default [
  {name: 'view', tag: 'div'},
  {name: 'inlineBlock', tag: 'div', styles: {display: 'inline-block'}},
  // etc...
]
```

Then in `index.js` we can reduce these to glamorousFactories with the styles pre-applied:

```
(comps, {name, tag, styles}) => {
  const capitalName = capitalize(name)
  comps[capitalName] = glamorous[tag](styles)
  // ..etc
}
```

I'm also thinking that it may be better to not put them directly on `glamorous` but instead scope them to a property on `glamorous`. Maybe `glamorous.display`? Because they're all display-specific? So you'd do:

```javascript
const {View, InlineBlock} = glamorous.display

// use them here
```

Thoughts?